mwdb/schema/object.py

```diff
@@ -107,6 +107,9 @@ class ObjectItemResponseSchema(Schema):
     children = fields.Nested(
         ObjectListItemResponseSchema, many=True, required=True, allow_none=False
     )
+    attributes = fields.Nested(
+        AttributeItemResponseSchema, many=True, required=True, allow_none=False
+    )
```

Comment:
That's bad idea. Attributes should be gathered via `get_attributes` because direct access to attribute will get all object attributes regardless of user permissions.
```suggestion
```

Reply:
Of course. All object attributes were visible.
A post_dump was added to replace object attributes with only accessible by current logged user.